sapl/templates/painel/index.html

```diff
@@ -124,24 +123,40 @@ <h2 class="text-subtitle">Tema da Sessão Solene</h2>
   {% endblock webpack_loader_chunks_js %}
 
   <script type="text/javascript">
-    var d = new Date();
-    var n = d.toLocaleDateString();
-    document.getElementById("date").innerHTML = n;
 
       $(document).ready(function() {
+
+         // As constantes decisões sobre a existência ou não do horário de verão,
+         // assim como que data começa ou termina, fizeram com que fosse necessário
+         // substituir a chamada a Date() por um esquema mais elaborado, onde se
+         // recupera o offset do UTC (-3 GMT, no caso de Brasília) e seta-se
+         // manualmente. Esta informação vem do servidor, desta forma não ficamos
+         // na dependência da atualização de browser, pois tanto o Date() em JS
+         // quanto as libs python (django.utils.timezone, datetime, pytz, etc)
+         // lêem do tzdata, que precisa ser atualizado toda vez que o governo
+         // brasileiro modifica alguma coisa relacionada ao horário de verão.
+         // Recuperando essa informação do servidor só teremos que atualizar as
+         // libs tzdata (Linux) e pytz (Python) uma vez. Além disso, o uso da
+         // biblioteca moment.js é recomendada, pois ela trata data e hora
+         // melhor que o Date() do JS.
+
+         $("#date").append(moment().format("DD/MM/YY"));
+
+         var utc_offset = $("#utc_offset").val()
```

Comment:
não precisa jogar num input e depois pegar... só isso abaixo é suficiente...
```suggestion
         var utc_offset = {{utc_offset}}
```

Reply:
Sem o parseFloat deu erro, talvez por conta da vírgula. Assim funcionou.
```
    var offset = parseFloat({{ utc_offset }});
```